boms/user/server/ee/pom.xml

```diff
@@ -366,7 +366,7 @@
                             <!-- include security: elytron  -->
                             <dependency>
```

Comment:
You also need all other components that you had, since those are the ones that are in WildFly.
PS: In my BOM review I suggested an artifactId change to wildfly-elytron-bom-all